sdk/src/fee.rs

```diff
@@ -80,17 +80,10 @@ impl FeeStructure {
     pub fn calculate_fee(
         &self,
         message: &SanitizedMessage,
-        lamports_per_signature: u64,
+        _lamports_per_signature: u64,
```

Comment:
will remove this parameter in separate PR involving removing fetching nonce lamports_per_signature.